### PR TITLE
chore: Cleanup for meta data files for the A380X

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary of Changes
Minor cleanup for A380X meta data

Discord username (if different from GitHub): cdr_maverick

## Testing instructions
not required

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/aircraft.cfg [hunk 1]
```diff
@@ -2,79 +2,77 @@
...
-icao_engine_type =Jet
-icao_engine_count =4
-icao_WTC =M
-icao_generic=0
+icao_type_designator = "A388"
+icao_manufacturer = "AIRBUS"
+icao_model = "A-380-800"
+icao_engine_type = Jet
+icao_engine_count = 4
+icao_WTC = H
```
Comment (tracernz) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/aircraft.cfg:20]: Does MSFS not support J (super)?

Comment (frankkopp) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/aircraft.cfg:20]: Idk - I looked at the 747 which uses H

**Review by tracernz — APPROVED** [review]

**Comment by trevorhannant** [comment]
Isn't the ICAO wake category for the A380 'J'?

https://skybrary.aero/articles/icao-wake-turbulence-category

**Comment by alepouna** [comment]
refer to the review comment

**Comment by frankkopp** [comment]
> Isn't the ICAO wake category for the A380 'J'?
> 
> [skybrary.aero/articles/icao-wake-turbulence-category](https://skybrary.aero/articles/icao-wake-turbulence-category)

As alenpouna commented above - I added this comment to the file:

; the A380-800 is the only WTC J aircraft but MSFS does not support J as far as we know
; @see https://skybrary.aero/articles/icao-wake-turbulence-category
icao_WTC = H

**Pull request merged**